video/out/opengl/context_x11egl.c

```diff
@@ -18,10 +18,14 @@
 #include <assert.h>
 
 #include <X11/Xlib.h>
-#include <X11/extensions/Xpresent.h>
```

Comment:
This header doesn't need to be here. Not sure why I put it here.

Reply:
removed